civic.json

```diff
@@ -0,0 +1,50 @@
+
+{
+    "id": "",
+    "name": "WABA Bike Infrastructure Map Project",
+    "status": "Beta",
+    "type": "Website",
+    "contact": {
+        "name": "Alex Ulsh",
+        "email": "alexandra.ulsh@gmail.com"
+    },
+    "partners": [
+        {
+            "url": "http://codefordc.org",
+            "name": "Code for DC"
+        },
+        {
+            "url": "http://www.waba.org/",
+            "name": "Washington Area Bicyclist Association"
+        }
+    ],
+    "description": "Creating a bike map advocacy tool for Washington Area Bicyclist Association",
+    "license": "",
```

Comment:
License is a *should* be included field [per the specification](http://open.dc.gov/civic.json/specification.html), let's remove from the `civic.json` until https://github.com/dcfemtech/hackforgood-waba-map/issues/52 is closed.

Once we decide on a license we should then add it to the `civic.json`.